src/main/java/uk/gov/hmcts/cmc/claimstore/services/document/DocumentManagementService.java

```diff
@@ -105,12 +115,7 @@ public ClaimDocument logUploadDocumentFailure(
         PDF pdf
     ) {
         String filename = pdf.getFilename();
-        String errorMessage = String.format(
-            "After retry : unable to upload document %s into document management due to %s",
-            filename, exception.getMessage()
-        );
-
-        logger.info(errorMessage, exception);
+        logger.info(exception.getMessage() + " " + exception.getCause(), exception);
```

Comment:
would be better if we keep this log as this confirms retry has been done

Reply:
Not sure what exactly you mean but the message is being logged in the retry block so I don't think we need to specify that explicitly. Also, The previous log message was logging  redundant message.